app/assets/javascripts/discourse/lib/dialog-holder/addon/components/dialog-holder.hbs

```diff
@@ -0,0 +1,35 @@
+<div id="dialog-holder" class="dialog-container {{this.dialog.classes}}" aria-labelledby={{this.dialog.titleElementId}} aria-hidden="true">
```

Comment:
maybe we use `dialog.ariaLabelledBy` as it's used only for this?

Reply:
I prefer to keep `titleElementId` because it's clearer that it relies on the title element (and when title element is missing, this is going to be empty).